src/foundry/foundry.js/applications/formApplication.d.ts

```diff
@@ -72,10 +73,14 @@ declare abstract class FormApplication<
   protected _render(force?: boolean, options?: Application.RenderOptions): Promise<void>;
 
   /**
-   * @param options - (unused)
    * @override
    */
-  protected _renderInner(data: D, options?: Application.RenderOptions): Promise<JQuery>;
+  protected _renderInner(data: D): Promise<JQuery>;
```

Comment:
Looks like this can also return `HTMLFormElement`, looking at the implementation

Reply:
ah, yeah, you are right, misread that.